### PR TITLE
Feature/cas 122

### PR DESCRIPTION
**Pull request by blingwang**

### CODE REVIEW

File: env [hunk 1]
```diff
@@ -26,10 +26,8 @@ PROJ="dex"
 ORG_PATH="github.com/coreos"
 REPO_PATH="${ORG_PATH}/${PROJ}"
 
-if [ ! -h gopath/src/${REPO_PATH} ]; then
```
Comment (thephred) [inline env]: Why remove the existing check?

File: static/fixtures/clients.json [hunk 2]
```diff
@@ -0,0 +1,7 @@
+[
+  {
+    "id": "castle-global-frontend",
+    "secret": "ZXhhbXBsZS1hcHAtc2VjcmV0",
```
Comment (thephred) [inline static/fixtures/clients.json]: We probably want to generate this during bring up of global and have it loaded from ENV or args to start script.

Comment (blingwang) [inline static/fixtures/clients.json]: The client id and secret are already generated during k8s deployment of dex. This fixture is only for running dex out of k8s, for testing purpose.

File: env (hunk 1 above)
Comment (blingwang) [inline env]: The symbolic link maybe exists, but linked to a wrong path. We have to do the linking for every build. If running build in docker container, the path should be linked to a file path different from running build locally.

Comment (blingwang) [inline env]: Also '-p' for 'mkdir' checks the existing of this dir.

Comment (thephred) [inline env]: The  mkdir -p will be ok  but not the ln -s.  I'm talking about repeatability.  If the target file of the ln -s already exists the command will fail with "File exists"

Comment (blingwang) [inline env]: Should be 'ln -sf'. The commit missed one change. Adding it now..

**Comment by thephred** [comment]
With the minor change mentioned below LGTM.

**Pull request merged**